src/main/resources/assets/appliedenergistics2/lang/de_de.lang

```diff
@@ -358,12 +384,38 @@ waila.appliedenergistics2.Unlocked=Entsperrt
 waila.appliedenergistics2.Showing=Zeigt
 waila.appliedenergistics2.Contains=Enthält
 waila.appliedenergistics2.Channels=%1$d von %2$d Kanälen
+waila.appliedenergistics2.P2PUnlinked=Link getrennt
+waila.appliedenergistics2.P2PInputOneOutput=Verbunden (Eingangsseite)
+waila.appliedenergistics2.P2PInputManyOutputs=Verbunden (Eingangsseite) - %d Ausgänge
+waila.appliedenergistics2.P2POutput=Verbunden (Ausgangsseite)
+ 
+// TheOneProbe
+theoneprobe.appliedenergistics2.crafting=Herstellen: %1$s
+theoneprobe.appliedenergistics2.device_online=Gerät verfügbar
+theoneprobe.appliedenergistics2.device_offline=Gerät nicht verfügbar
+theoneprobe.appliedenergistics2.device_missing_channel=Gerät fehlt Kanal
+theoneprobe.appliedenergistics2.locked=Gesperrt
+theoneprobe.appliedenergistics2.unlocked=Entsperrt
+theoneprobe.appliedenergistics2.showing=Zeigt
+theoneprobe.appliedenergistics2.contains=Enthält
+theoneprobe.appliedenergistics2.channels=%1$d von %2$d Kanälen
+theoneprobe.appliedenergistics2.p2p_unlinked=Link getrennt
```

Comment:
"Unlinked" means that the P2P device is not linked, in German that would be "Nicht verlinkt" and not "Link disconnected".